configure-repo.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+#
+# Setup git hooks in this or all repositories.
+#
+# Run this script once to configure your system to work with our Git workflow.
+#
+
+case $1 in
+  --global)
+    # Requires Git 2.9 or later.
+    coreHooksPath=`git config --global core.hooksPath`
+    if [[ -z "$coreHooksPath" ]]; then coreHooksPath="(not set)"; fi
```

Comment:
On my Linux box, I get the error
```
./configure-repo.sh: 13: ./configure-repo.sh: [[: not found
```

bash syntax?

Reply:
btw even with the error, the rest of the script ran

Reply:
hmm. I dunno. Which version of bash do you run? Should be recent I'd have thunk.

Reply:
https://serverfault.com/questions/52034/what-is-the-difference-between-double-and-single-square-brackets-in-bash

Reply:
Read much of that link and TIL. `[[` FTW.

Reply:
> btw even with the error, the rest of the script ran

Oops, should have `set -e` at the start of the script!